### ISSUE TITLE
Question about voting and the final private score

### ISSUE THREAD

**Issue by mcps5601** [qqandy0120/AICUP-2023-Team-3455#1]
https://github.com/qqandy0120/AICUP-2023-Team-3455/blob/636ed5f8133b5c1965678dbe9377a636fd06c9a8/README.md?plain=1#L218

Was the final score achieved with voting or without voting?

**Comment by zincjian** [comment]
Hi, 
Thanks for the question! The final score is achieved with the voting with three models which we provided in the [Subtask 3][Step 1]. 

Due to those models were fine-tuned on the same pretrained model, you may place them in the same directory and set the `--do_ensemble` option to `1` and `--do_ensemble_topk` option to `3` in `claim_vrf.py` to do the label voting, too (in the default code we provided). Besides, `json_voting.py` is implemented for the label voting for output files of different pretrained models.
